setup.py

```diff
@@ -137,20 +137,13 @@ def initialize_options(self) -> None:
 )
 
 airflow_dependencies = [
-    # Minimum version here is needed to avoid syntax error in setup.py
-    # in 1.10.0
-    'apache-airflow>=1.10.1,<2'
+    'apache-airflow>=2',
+    'apache-airflow-providers-amazon',
+    'apache-airflow-providers-google',
 ]
 
 db_dependencies = [
-    # Lower bound (>=1.3.18) is to improve package resolution performance
-    #
-    # Upper bound (<1.4) is to avoid 1.4 which has breaking changes and is
-    # incompatible with python-bigquery-sqlalchemy per
-    # https://github.com/googleapis/python-bigquery-sqlalchemy/issues/83
-    # Can lift this once records-mover itself is compatible and
-    # other packages have appropriate restrictions in place.
-    'sqlalchemy>=1.3.18,<1.4',
+    'sqlalchemy>=1.3.18',
```

Comment:
Looking at the removed comment, we required sqlalchemy < 1.4 because of bigquery compatibility issues. Given our bigquery integration tests are currently disabled, how has this change been tested?

Reply:
@Brunope Re-added the BQ tests, if this passes checks we should be good to approve!